index.php

```diff
@@ -1,15 +1,9 @@
 <?php
-
-define('ROOT_DIR', realpath(dirname(__FILE__)) . '/');
-define('LIB_DIR', ROOT_DIR . 'lib/');
-define('VENDOR_DIR', ROOT_DIR . 'vendor/');
-define('PLUGINS_DIR', ROOT_DIR . 'plugins/');
-define('THEMES_DIR', ROOT_DIR . 'themes/');
-define('CONFIG_DIR', ROOT_DIR . 'config/');
-define('CACHE_DIR', LIB_DIR . 'cache/');
-
-define('CONTENT_EXT', '.md');
-
-require_once(VENDOR_DIR . 'autoload.php');
-require_once(LIB_DIR . 'pico.php');
-$pico = new Pico();
+require_once(__DIR__ . '/vendor/autoload.php');
+$pico = new Pico(
+    __DIR__,
+    'config/',
```

Comment:
I wonder if we could move configuration directly in index.php, instead of maintaining a separate configuration file (or at least code the configuration retrieval here).

Let me elaborate. I'm using Pico within a Silex installation, redirecting all calls to a specific pattern to the Pico->run function. Thing is Silex has already its configuration system, and having Pico duplicates a bit the logic (I need to maintain/deploy one file for it).

Something cool would be:
```php
// index.php
$confArray = array(
    'someconf'=>array()
);
$engine = new Pico($confArray);
$engine->run();
```
... but of course it would oblige us to remove index.php from the repo, provide tempalte for ti and make things more complicated in the end.

One way to mitigate this would be to make the Pico constructor aware, if $config is array then I load it instead of fetching file.

Of course, this is a new feature and it would be totally ok to postpone it until 1.0 is merged. I can provide PR if the feature sounds usefull to you guys. @theshka @PhrozenByte ?